### PR TITLE
messaging: Add new modules for kafka-configs

### PR DESCRIPTION
**Pull request by gdelpierre**
##### SUMMARY
This module aims to allow a user to manage Kafka configs  

Internally it relies on kafka scripts (kafka-configs.sh).

##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

 - kafka_configs.py

##### ANSIBLE VERSION
```
ansible 2.3.1.0
  config file = /home/gdelpierre/devel/roles/ansible.cfg
  configured module search path = [u'./library/']
  python version = 2.7.13 (default, Jan 19 2017, 14:48:08) [GCC 6.3.0 20170118]
```

### CODE REVIEW

**Comment by gdelpierre** [comment]
cc @Spredzy @pilou- @mscherer @jwoodrich

**Comment by gdelpierre** [comment]
@jwoodrich I've changed the way to deal with configs according to your feedback on #27323 .
Could you test it ?

**Comment by mattclay** [comment]
CI failure in unit tests: https://app.shippable.com/github/ansible/ansible/runs/37919/2/tests

File: lib/ansible/modules/messaging/kafka_configs.py [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+            - Kafka executable path if not in your current PATH or if it name differs.
+    zookeeper:
+        required: True
+        description:
+            - The connection string for the zookeeper connection.
+              Multiple connection strings are allowed.
+'''
+
+EXAMPLES = '''
+name: 'list config'
```
Comment (gundalow) [inline lib/ansible/modules/messaging/kafka_configs.py:53]:
```suggestion
- name: 'list config'
```

Same for others. You will need to check indentation

**Comment by Akasurde** [comment]
Please submit a new module request to https://github.com/ansible-collections/community.general. Thanks for your contribution.

**Pull request closed without merging**